RELEASE.md

```diff
@@ -0,0 +1,14 @@
+# Release Process
+
+1. Merge PR.
```

Comment:
Either here after step one or after step 3, it would be good to suggest the user runs the full acceptance test suite against the master branch.

Reply:
Ah yes good shout!